lib/event.rb

```diff
@@ -0,0 +1,29 @@
+
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name,ages)
+    @name = name
+    @ages = ages
+    @sorted_ages = ages.sort
```

Comment:
Don't make a new instance variable for sorted_ages. If you need the sorted ages, you can call @ages.sort. For instance, in your max_age method, you can just the method body to `@ages.sort.last`. In general avoid using instance variables unless absolutely necessary, when you need the data to persist through the object's lifetime. A good sign you don't need an instance variable is if it can be easily calculated from another instance variable.